versions.tf

```diff
@@ -2,6 +2,9 @@ terraform {
   required_version = ">= 0.14, < 2.0"
 
   required_providers {
-    google-beta = "~> 3.75"
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 3.75, < 5.0"
+    }
```

Comment:
there are references in code to google-beta that need to be removed

Reply:
unless it fails for still being beta

Reply:
> there are references in code to google-beta that need to be removed

hey @mariux, from https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/artifact_registry_repository we get

> This resource is in beta, and should be used with the terraform-provider-google-beta provider. See Provider Versions for more details on beta resources.

and they state (https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_versions#using-the-google-beta-provider) that we should keep using the `google-beta` provider as in https://github.com/mineiros-io/terraform-google-artifact-registry-repository/blob/main/main.tf#L2

Reply:
oh I think I got it know :sweat_smile: